lib/tracking/google-analytics.js

```diff
@@ -1,17 +1,27 @@
-
 const buttonTrackingAttributes = (buttonText) => {
-    return JSON.stringify({
-      event_name: "select_content",
-      Type: "tabs",
-      text: buttonText,
-      external: "false",
-      method: "primary click",
-      section: document.title,
-      action: "select",
-      tool_name: "Visual Editor",
-    });
-}
+  return JSON.stringify({
+    event_name: "select_content",
+    Type: "tabs",
+    text: buttonText,
+    external: "false",
+    method: "primary click",
+    section: document.title,
+    action: "select",
+    tool_name: "Visual Editor",
+  });
+};
 
 export const trackButton = (button) => {
   button.setAttribute("data-ga4-event", buttonTrackingAttributes(button.title));
-}
\ No newline at end of file
+};
+
+export const dispatchSelectTrackingEvent = (select) => {
+  select.dispatchEvent(
```

Comment:
I'm wondering if we should dispatch these on the visual editor `div` rather than the select.
It feels like this shouldn't be exposed to consumers of the event.
At the same the select is here and available so I don't want to complicate the code more than is necessary.

Reply:
Cool cool - The event bubbles, so I thought it didn't matter too much what element it was emitted on as any parent can listen for it.